include/language-factory.php

```diff
@@ -98,6 +98,31 @@ public static function get_from_terms( array $terms ) {
 		$flag_props = self::get_flag( $data['flag_code'], $data['name'], $data['slug'], $data['locale'] );
 		$data       = array_merge( $data, $flag_props );
 
+		$static_pages = array(
+			'page_on_front'  => 0,
+			'page_for_posts' => 0,
+		);
```

Comment:
Is it really needeed as you made these properties optional?

Reply:
See @Hug0-Drelon answser https://github.com/polylang/polylang/pull/1195#discussion_r1069549848.
However pass an empty array to the filter as suggested by @chouby in his https://github.com/polylang/polylang/pull/1195#discussion_r1071013645

Done in d7a53cc35ac9e50030dbc56dc127071451c9fb34